pyproject.toml

```diff
@@ -38,8 +38,8 @@ iceberg = ["icebridge"]
 ray = [
   # Inherit existing Ray version. Get the "default" extra for the Ray dashboard.
   "ray[data, default]>=2.0.0",
-  # Ray has a bug with grpcio 1.52.0: https://github.com/ray-project/ray/issues/32246
-  "grpcio<1.52.0"
```

Comment:
@jaychia Is this grpcio issue no longer a factor?

Reply:
Ah looks like this was a driveby cleanup, just found this commit! https://github.com/Eventual-Inc/Daft/commit/249da5a95cd21966c7f1f68806b6701d123f3f92